### ISSUE TITLE
Modifying Ray config variables

### ISSUE THREAD

**Issue by aksld** [Pierre-Lannoy/wp-decalog#56]
I'm trying to use DecaLog on my development environment (docker container). Despite setting up the Ray logger, no events arrive on Ray. 

Perhaps you should allow variables to be modified or use the [ray.php](https://spatie.be/docs/ray/v1/environment-specific-configuration/docker) ? Or a better solution? ;)

```
protected $defaultSettings = [
        'enable' => true,
        'host' => 'host.docker.internal',
        'port' => 23517,
        'remote_path' => null,
        'local_path' => null,
        'always_send_raw_values' => false,
    ];
```

**Comment by Pierre-Lannoy** [comment]
Hello Axel!
Of course, you're right: for now Ray settings are fixed.
I will refactor the logger, so that it can take into account not only generic settings.
Thanks for the suggestion…

**Comment by Pierre-Lannoy** [comment]
Hello Axel!
It will be available in 3.10.0 release (expected release in some days).
